src/invoices/InvoiceForm/CurrentExchangeRate/CurrentExchangeRate.test.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { act, cleanup, render, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+
+import '@folio/stripes-acq-components/test/jest/__mock__';
+
+import CurrentExchangeRate from './CurrentExchangeRate';
+
+const renderCurrentExchangeRate = (
+  exchangeFrom,
+  exchangeTo,
+  mutator,
+) => (render(
+  <IntlProvider locale="en">
+    <CurrentExchangeRate
+      label={<div>Label</div>}
+      exchangeFrom={exchangeFrom}
+      exchangeTo={exchangeTo}
+      mutator={mutator}
+      setExchangeRateRequired={jest.fn}
+      setExchangeRateEnabled={jest.fn}
+    />
+  </IntlProvider>,
+));
+
+describe('CurrentExchangeRate component', () => {
+  let mutator;
+
+  beforeEach(() => {
+    mutator = {
+      exchangeRate: {
+        GET: jest.fn(),
+      },
+    };
+  });
+
+  afterEach(cleanup);
```

Comment:
it's not required AFAIK https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup